containers/allan_dip_bot/run_bot.py

```diff
@@ -26,6 +27,7 @@
 from baseline_bots.bots.random_proposer_bot import RandomProposerBot_AsyncBot
 from baseline_bots.bots.smart_order_accepter_bot import SmartOrderAccepterBot
 
+Aggressiveness = Enum("Aggressiveness", ["aggressive", "moderate", "friendly"])
```

Comment:
Import this from `smart_order_accepter_bot.py` instead of duplicating the definition. I'm not even sure the redefinition would work properly.

Reply:
done